src/ReactSelectize.js

```diff
@@ -31,6 +31,7 @@
       groupsAsColumns: false,
       highlightedUid: undefined,
       inputProps: {},
+      listboxId: '',
```

Comment:
can you duplicate this in `src/SimpleSelect` to remove the dependency on uuid?

Reply:
this prop here was removed in favor of always using a generated id